Documentation/About/Index.rst

```diff
@@ -24,14 +24,15 @@ Compatibility
 =============
 
 .. csv-table:: Version Matrix
-   :header: "Extension Version", "TYPO3 v10 Support", "TYPO3 v9 Support", "TYPO3 v8 Support"
+   :header: "Extension Version", "TYPO3 v11 Support", "TYPO3 v10 Support", "TYPO3 v9 Support", "TYPO3 v8 Support"
    :align: center
 
-        "5.0.x", "🙋‍♂️", "🙅‍♀️", "🙅‍♀️"
-        "4.2.x", "🙋‍♂️", "🙋‍♂️", "🙅‍♀️"
-        "4.1.x", "🙅‍♀️", "🙋‍♂️", "🙋‍♂️"
-        "4.0.x", "🙅‍♀️", "🙋‍♂️", "🙋‍♂️"
-        "3.1.x", "🙅‍♀️", "🙋‍♂️", "🙋‍♂️"
+        "5.2.x", "🙋‍♂️", "🙋‍♂️", "🙅‍♀️", "🙅‍♀️"
+        "5.0.x", "🙅‍♀️", "🙋‍♂️", "🙅‍♀️", "🙅‍♀️"
+        "4.2.x", "🙅‍♀️", "🙋‍♂️", "🙋‍♂️", "🙅‍♀️"
+        "4.1.x", "🙅‍♀️", "🙅‍♀️", "🙋‍♂️", "🙋‍♂️"
+        "4.0.x", "🙅‍♀️", "🙅‍♀️", "🙋‍♂️", "🙋‍♂️"
+        "3.1.x", "🙅‍♀️", "🙅‍♀️", "🙋‍♂️", "🙋‍♂️"
```

Comment:
Can the "funny" icons be replaced by "x" and "-"?

Reply:
Changed icons